packages/screenshots/testplane/test.mjs

```diff
@@ -5,7 +5,6 @@ import filenamify from 'filenamify';
 
 import Actions from './actions.js';
 
-// eslint-disable-next-line import/no-unresolved
```

Comment:
this is a generated file that's absent until you run visual tests

Reply:
So I should delete it? Sure, thanks!

Reply:
Yes, and it means that we still need a eslint-disable here

Reply:
Fun part is that one have to hack this around : D
![2024-12-20 16 20 23](https://github.com/user-attachments/assets/2e275055-407d-4b4b-93b1-a4357902da28)